src/samplers/remote_sampler.js

```diff
@@ -93,11 +101,14 @@ export default class RemoteControlledSampler {
             let maxTracesPerSecond = strategy.rateLimitingSampling.maxTracesPerSecond;
             newSampler = new RateLimitingSampler(maxTracesPerSecond);
         } else {
+            this._metrics.samplerParsingFailure.increment(1);
             this._logger.error('Unrecognized strategy type: ' + JSON.stringify({error: strategy}));
```

Comment:
`JSON.stringify({error: strategy})` - should our logging API be key-value based instead, so that we don't need to stringify fields?  I thought that's what logtron supports.